src/util.rs

```diff
@@ -228,3 +230,74 @@ pub fn short_file_path<'a>(target: &'a str, current_dir: &str) -> &'a str {
         .and_then(|p| p.to_str())
         .unwrap_or(target)
 }
+
+/// Given a starting Location, try to find `name` in a file and report its Position
```

Comment:
I don't think we should add more to `src/util.rs`, because it's such an opaque name. Rather add these where needed and *maybe* extract them when we want to reuse them.

Reply:
Done. Moved into `document_symbol.rs` (which also now contains everything from `kak_lsp_custom.rs`